daemon/container_windows.go

```diff
@@ -18,6 +18,8 @@ import (
 // the container. Docker has no context of what the default path should be.
```

Comment:
I'm a little surprised golint doesn't pickup that the godoc doesn't start "DefaultPathEnv...."